src/main/java/frc/robot/Constants.java

```diff
@@ -51,9 +51,9 @@ public final class Constants {
   public static final double ANGLETOLERANCE = 1;
 
   // Auto Leveling
-  public static final double FORWARD_VELOCITY = .05;
-  public static final double STRAFE_VELOCITY = .03;
-  public static final double ANGLE_TOLERANCE = 1.5;
+  public static final double FORWARD_VELOCITY = .1;
+  public static final double STRAFE_VELOCITY = .01;
```

Comment:
Now that we solved the issue of the command not seeming to work (too slow) should we bring strafing for auto balance back?

Reply:
Yeah we could. But idk if we really need it technically need it since we can move on the charging station without slipping. But we can try it